### DISCUSSION TITLE
[Unstable] Scrap Savant error message

### DISCUSSION THREAD

**Discussion by ubertpendragon** [FakeFishGames/Barotrauma#14003] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

![image](https://github.com/FakeFishGames/Barotrauma/assets/73229309/b45fdb86-35a4-4394-a71e-ccd4c58e0bd2)

### Reproduction steps

`givetalent "Scrap Savant"`

### Bug prevalence

Just once

### Single player or multiplayer?

Single player

### Version

v1.4.6.0 (Blood in the Water Update, hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by Jasontti** [comment]
Tested the fix for this (https://github.com/FakeFishGames/Barotrauma-development/pull/5331/commits/0282c45e73ba92b067109a01e68e6fdb6597a967) and found no issues.
Closing discussion as resolved.
